### DISCUSSION TITLE
Disney Junior [IN]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#904] [Q&A]
### Channel ID

DisneyJunior.in

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/in.m3u#L167-L168
